app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -9,51 +9,52 @@
 
     <Button
         android:id="@+id/btn_back"
-        android:layout_width="50dp"
-        android:layout_height="50dp"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintHorizontal_bias="0.008"
+        android:layout_width="@dimen/dp50"
+        android:layout_height="@dimen/dp50"
         android:background="@drawable/btn_icon_back"
         android:backgroundTint="@color/secondary"
-        />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.008"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="SpeakableTextPresentCheck" />
     <ImageView
         android:id="@+id/img_logo"
-        android:layout_width="139dp"
-        android:layout_height="70dp"
+        android:layout_width="@dimen/signUp_logo_139dp"
+        android:layout_height="@dimen/signUp_logo_70dp"
         android:src="@drawable/ic_sign_logo"
+        android:contentDescription="@string/signUp"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toBottomOf="@id/btn_back"
         app:layout_constraintHorizontal_bias="0.2"
         />
     <TextView
         android:id="@+id/txt_signUp"
-        android:layout_width="250dp"
-        android:layout_height="35dp"
+        android:layout_width="@dimen/signUp_250dp"
+        android:layout_height="@dimen/signUp_35dp"
         app:layout_constraintTop_toBottomOf="@id/img_logo"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.34"
         android:text="@string/create"
-        android:layout_marginTop="28dp"
+        android:layout_marginTop="@dimen/signUp_28dp"
         android:textColor="@color/quaternary"
-        android:textSize="25sp"
+        android:textSize="@dimen/signUp_25sp"
```

Comment:
remember not to tie dimens to specific screens when they can be reusable

Reply:
i'll implement it, thanks